### PR TITLE
This is a better solution the the SFLogger crash. (As it works)

### PR DESCRIPTION
**Pull request by sgoldberg-sfdc**
- Unit tests are also updated.

### CODE REVIEW

**Comment by sgoldberg-sfdc** [comment]
@bhariharan @khawkins @huminzhi

**Comment by sgoldberg-sfdc** [comment]
No comment @bhariharan . Literally, a comment would break those two tests.

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/Logging/SFLogger.m [hunk 1]
```diff
@@ -230,11 +230,11 @@ + (NSString*)loggingIdentifier {
 }
 
 - (void)log:(SFLogLevel)level msg:(NSString *)msg {
-    [self log:level identifier:[self.class loggingIdentifier] format:msg, nil];
+    [self log:level identifier:[self.class loggingIdentifier] format:@"%@", msg];
```
Comment (khawkins) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/Logging/SFLogger.m]: My head was circling around this solution, but I hadn't had time to focus on it.  I felt a one-liner was a distinct possibility.  👍

**Comment by bhariharan** [comment]
test me

**Comment by bhariharan** [comment]
test me

**Comment by huminzhi** [comment]
test me

**Pull request merged**